package.json

```diff
@@ -36,7 +36,7 @@
   "homepage": "https://github.com/dlmr/react-router-redial#readme",
   "peerDependencies": {
     "react": "*",
-    "react-router": "^2.4.0",
+    "react-router": "^3.0.0",
```

Comment:
I don't think we should _force_ `react-router@v3`, since we work just as well with `react-router@v2`. So this should change to `^2.4.0 || ^3.0.0`